projects/core/src/auth/http-interceptors/auth-error.interceptor.ts

```diff
@@ -62,9 +62,8 @@ export class AuthErrorInterceptor implements HttpInterceptor {
                   errResponse.url.indexOf(OAUTH_ENDPOINT) !== -1 &&
                   errResponse.error.error === 'invalid_token'
                 ) {
-                  return of(
-                    this.userErrorHandlingService.handleExpiredRefreshToken()
-                  );
+                  this.userErrorHandlingService.handleExpiredRefreshToken();
```

Comment:
can we really invoke this function here and then return an empty observable?

Reply:
that function returns undefined, so yes of course